cpp/src/slice2js/Gen.cpp

```diff
@@ -510,47 +489,43 @@ Slice::JsVisitor::writeConstantValue(
     return os.str();
 }
 
-StringList
-Slice::JsVisitor::splitComment(const ContainedPtr& p)
```

Comment:
After refactoring, this function was only used by `writeDocCommentFor` (the function underneath it). So I just inlined it into that function.